orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/testframework/utils/CucumberProfileReader.java

```diff
@@ -51,7 +50,7 @@ public Profile read() {
 
     private Profile getProfile() {
         try {
-            MainConstraintReader constraintReader = new MainConstraintReader(constraintReaderMap);
+            MainConstraintReader constraintReader = new MainConstraintReader(constraintReaderMap, new AtomicConstraintValueReader(null));
```

Comment:
can we not instantiate a FromFileReader and use it sinteda of null - also means if ever read set from file in our cucumber test it will work rather than give us a confusing null ptr exception